operator/pkg/remote/kyma_synchronization_context.go

```diff
@@ -286,3 +290,24 @@ func GetRemoteObjectKey(kyma *v1alpha1.Kyma) client.ObjectKey {
 	}
 	return client.ObjectKey{Namespace: namespace, Name: name}
 }
+
+func (c *KymaSynchronizationContext) InsertWatcherLabels(ctx context.Context, remoteKyma *v1alpha1.Kyma) error {
+	recorder := adapter.RecorderFromContext(ctx)
+
+	ownedByValue := fmt.Sprintf("%s__%s", c.ControlPlaneKyma.Namespace, c.ControlPlaneKyma.Name)
+	watchedByValue := "lifecycle-manager"
+
+	if remoteKyma.Labels == nil {
+		remoteKyma.Labels = make(map[string]string)
+	}
+
+	remoteKyma.Labels["operator.kyma-project.io/owned-by"] = ownedByValue
+	remoteKyma.Labels["operator.kyma-project.io/watched-by"] = watchedByValue
+
+	err := c.RuntimeClient.Update(ctx, remoteKyma)
+	if err != nil {
+		recorder.Event(c.ControlPlaneKyma, "Warning", err.Error(), "could not update runtime kyma labels")
+		return err
+	}
```

Comment:
```suggestion
```
Please do not introduce an update into the remote SKR when you can simply update the object and then let it be updated in https://github.com/kyma-project/lifecycle-manager/pull/182/files#diff-bc1bb91f1f01617c2d3693314155dcfad6e5a612788e40ec921f8236d0a5a11cR242-R244 right after your method.
This introduces a second roundtrip to SKR otherwise

Reply:
^^ @jeremyharisch  I thought this is exactly what we discussed: consolidating both requests into 1 update call